leakcanary-analyzer/src/main/java/com/squareup/leakcanary/AnalysisResult.java

```diff
@@ -59,7 +61,8 @@ public static AnalysisResult failure(Throwable failure, long analysisDurationMs)
 
   /**
    * The number of bytes which would be freed if all references to the leaking object were
-   * released. 0 if {@link #leakFound} is false.
+   * released. {@link #RETAINED_HEAP_UNKNOWN} if the retained heap size was not computed. 0 if
```

Comment:
nit: UNKNOWN or IGNORE.  I feel like the latter conveys the disabling part more clearly.

Reply:
The only thing is that this is a result value, not a request value. ie it's not "please ignore" it's "we didn't compute it".

Reply:
so then IGNORED?  UNKNOWN is more like "we didn't know how to" compute it.

Reply:
Agree. It's just that "ignored" still feels not quite right to me. Maybe "skipped" ?